src/components/TransactionsTable/style.scss

```diff
@@ -216,8 +209,58 @@
       text-align: center;
     }
 
+    .spinner_container {
+      margin-bottom: 2000px;
```

Comment:
Strange value for margin. Why are you using need it?

Reply:
To make scroll persistent during loading